doc/ref/modules/mod_seo_sitemap.rst

```diff
@@ -9,6 +9,59 @@ the Google bot to index your site. By default, its generated
 sitemap.xml lists all pages of the categories text, event, location,
 collection and person in the system which are publicly viewable.
 
+The sitemaps follow the specification at https://www.sitemaps.org/protocol.html
+
+Maintaining the sitemap index
+-----------------------------
+
+The ``seo_sitemap`` model maintains an index of all pages (and their URLs) that are publicly readable.
+
+There is a button in the System > Status menu to rebuild the sitemap index. This is needed if there
+are (big) changes in the access control affecting the publicly viewable pages or if languages are
+changed.
```

Comment:
Would it be problematic or is it difficult to cover these cases automatically? To schedule a rebuild when any ACL or language setting are changed (theoretically, a change there yields a set of affected Resources, from which the required diff on the sitemap can be derived, but we would need an incremental sitemap build infrastructure for that).

Reply:
Language changes are now quite well covered (at least in the master version).

The ACL changes I am not so sure about, as going through 1M records because _something_ changed might be a bit much...  and indeed, we don't have a method to reverse engineer what records might be affected by a change (categories, content groups, etc.)

Reply:
Yeah indeed, it seems good for now, at least good to have discussed it here, the manual reindex button is needed as a fallback anyway.